src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -704,4 +706,9 @@
     <Reference Include="System.Security.Cryptography.OpenSsl" />
     <Reference Include="System.Security.Cryptography.Primitives" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Update="D:\repos\corefx\src\System.Net.Http\src/Resources/Strings.resx">
```

Comment:
I don't think this EmbeddedResource statement should be checked in since it refers to your local repo filepath.

Reply:
Looks like I didn't push the commit where I noticed this and removed it.  I agree.